### DISCUSSION TITLE
It would be amazing if Epic Stack supported "anonymous" sign-ins out of the box.

### DISCUSSION THREAD

**Discussion by grunklejp** [epicweb-dev/epic-stack#752] [Ideas]
Would this be something other people might want? I am considering adding this to my own app. I'll post back here on any progress I make. 

If anyone has any ideas or thoughts that might prove useful on my journey please let me know!

See: https://supabase.com/docs/guides/auth/auth-anonymous

**Comment by kentcdodds** [comment]
I'm sure someone would be interested! An [example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md) would be welcome!
